Editor/Resources/EditorWindow/Components/InfoLinks.uss

```diff
@@ -0,0 +1,14 @@
+﻿/* Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved. */
+/* SPDX-License-Identifier: Apache-2.0 */
+
+:root {
+    color: #4C7EFF;
+}
+
+.link-divider {
```

Comment:
To clarify, the difference between this and `divider` is it's vertical correct? Can it be `divider--vertical` then? No need to move it to `common.uss` because I don't think this'll be used elsewhere

Reply:
renamed